automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspaceDataPage.scala

```diff
@@ -50,22 +53,37 @@ class WorkspaceDataPage(namespace: String, name: String)(implicit webDriver: Web
     * @param downloadPath the directory where the browser saves downloaded files
     * @return the relative path to the moved download file, or None if downloadPath was not given
     */
-  def downloadMetadata(downloadPath: Option[String] = None): Option[String] = {
+  def downloadMetadata(downloadPath: Option[String] = None): Option[String] = synchronized {
 
     def archiveDownloadedFile(sourcePath: String): String = {
+      // wait up to 5 seconds for file exist
+      val f = new File(sourcePath)
+      logger.info(s"Retrying - downloaded file wait for exist: $f")
+      retry[Boolean](1.seconds, 10.seconds) {
```

Comment:
This will not raise an error if it times out.  Instead of retry do:
```
eventually {
  assert(f.exists(), s"Timed out waiting for file $f")
}
```